Core/Services/UserService.cs

```diff
@@ -26,4 +33,13 @@ public async Task<User> CreateUser(RegisterUserDto registerUserDto)
     {
         return await userRepository.GetUserByEmail(email);
     }
+
+    public async Task<GetUserDto?> UpdateUser(UpdateUserDto updateUserDto)
+    {
+        if (updateUserDto.Base64Image != null)
+        {
+            updateUserDto.BlobUrl = await blobStorageService.SaveImageToBlobStorage(updateUserDto.Base64Image, updateUserDto.UserEmail, null);
```

Comment:
when updating the profile image, make sure to pass in the previous blob URL, so It can be overwritten